documentation/api/metrics/v1/index.markdown

```diff
@@ -1,15 +1,13 @@
 ---
```

Comment:
Another doc change that I forgot ... should be minor, but the current versioning policy might want to be slightly modified to make sure the metrics versioning is clear: https://docs.puppetlabs.com/puppetdb/2.2/versioning_policy.html ... it should align with the HTTP API policy more or less, so I doubt you need to double up, just mentioning it covers the scope of the metrics stuff also might be enough.